### PR TITLE
MCSC-795 Clear Up No Longer in Use Msg Files

### PR DESCRIPTION
**Pull request by TJSarno**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -645,10 +622,6 @@ thresholdIncome.change.hidden.2021 = if your threshold income more than £200,00
 thresholdIncome.change.hidden.2022 = if your threshold income more than £200,000
 thresholdIncome.change.hidden.2023 = if your threshold income more than £200,000
 
-valueNewLtaCharge.title = What is the value of the new lifetime allowance charge?
-valueNewLtaCharge.heading = What is the value of the new lifetime allowance charge?
-valueNewLtaCharge.hint = Amounts must be rounded down to the nearest pound, for example 2300.
```
Comment (MichaelAAbbott) [inline conf/messages.en:650]: Does these definitely need removing?

File: conf/messages.en [hunk 2]
```diff
@@ -859,8 +822,6 @@ quarterChargePaid.title = In which quarter of that year was your lifetime allowa
 quarterChargePaid.heading = In which quarter of that year was your lifetime allowance charge paid?
 quarterChargePaid.hint = Select the quarter of the year your lifetime allowance charge was paid
 quarterChargePaid.aprToJul = 1 April to 30 June
-quarterChargePaid.julToOct = 1 July to 30 September
-quarterChargePaid.octToJan = 1 October to 31 December
```
Comment (MichaelAAbbott) [inline conf/messages.en:863]: Does these definitely need removing?

File: conf/messages.en [hunk 3]
```diff
@@ -548,10 +528,7 @@ pensionSchemeInputAmounts.change.hidden = pension scheme {0} input amounts
 whichScheme.title = Which scheme are you entering details for?
 whichScheme.heading = Which scheme are you entering details for?
 whichScheme.hint = Select from the options below or enter a new scheme.
-whichScheme.New = Enter a new pension scheme
```
Comment (MichaelAAbbott) [inline conf/messages.en:551]: Double check this please :)

File: conf/messages.en [hunk 4]
```diff
@@ -258,7 +249,6 @@ pIAPreRemedy.error.nonNumeric.2012 = Your pension input amount from 6 April 2011
 pIAPreRemedy.error.nonNumeric.2013 = Your pension input amount from 6 April 2012 to 5 April 2013 must only include numbers 0 to 9
 pIAPreRemedy.error.nonNumeric.2014 = Your pension input amount from 6 April 2013 to 5 April 2014 must only include numbers 0 to 9
 pIAPreRemedy.error.nonNumeric.2015 = Your pension input amount from 6 April 2014 to 5 April 2015 must only include numbers 0 to 9
-pIAPreRemedy.error.minimum.2011 = Your pension input amount from 6 April 2010 to 5 April 2011 must be £0 or more
```
[the remark this answers is not in the document]
Comment (TJSarno) [reply conf/messages.en:261]: readded as date is likely used

File: conf/messages.en (hunk 2 above)
Comment (TJSarno) [reply conf/messages.en:863]: removed as not in use and we suspect it was previously half removed from our service

File: conf/messages.en (hunk 3 above)
Comment (TJSarno) [reply conf/messages.en:551]: removed as not in use, bug raised to address the welsh translation

**Review by MichaelAAbbott — APPROVED** [review]

**Pull request merged**